README.md

```diff
@@ -12,13 +12,21 @@ First to install hugo simply run `brew install hugo v0.20.7` or `brew install hu
     $ npm run webpack-watcher
 ```
 
+The page will then be available at http://localhost:1313.
 
 ## Netlify & HUGO & Contentful & Github
+
 ![High level overview](https://github.com/shinetext/torch/blob/master/images/example.png)
 
+In order to pull content from staging to develop against, you can run:
 
-## About
+```
+$ CONTENTFUL_SPACE_ID=<contentful space id here> \
```

Comment:
do you think we should add a `.env` and update the readme to `source .env`?

Reply:
Oh interesting, I didn't think about using `source`. I can update the README to make note of that.

I'm hesitant to use the `.env` filename though. I feel like the expectation with that file is that it'd also be used by the application when it's running. But in this case, it's just for a couple npm scripts. And I'm pretty sure Hugo right now wouldn't actually be using it.